pkg/actions/commands.go

```diff
@@ -659,6 +659,17 @@ func Commands() {
 			case loglevel == "trace":
 				logr.SetLevel(logr.TraceLevel)
 				break
+			case loglevel == "debug":
+				logr.SetLevel(logr.DebugLevel)
+				break
+			case loglevel == "fatal":
+				logr.SetLevel(logr.FatalLevel)
+				break
+			case loglevel == "error":
+				logr.SetLevel(logr.ErrorLevel)
+				break
+			default:
+				logr.SetLevel(logr.InfoLevel)
 			}
 		} else {
```

Comment:
If you set the `value` of the flag to Info level for logging, you _shouldnt_ need this `if != "" / else` clause as it will take that as the default if the flag is not mentioned